src/dataframerow/utils.jl

```diff
@@ -4,11 +4,11 @@
 struct RowGroupDict{T<:AbstractDataFrame}
     "source data table"
     df::T
-    "number of groups"
+    "number of groups (i.e. highest group index: some groups may be empty)"
```

Comment:
nonempty group?

Reply:
I'm not sure I understand the question. Some of the "groups" counted here may be empty, so that just gives a value guaranteed to be as high or higher than the actual number of groups. (Actually this field doesn't seem to be used, so we could remove it.)

Reply:
I asked if we count only non-empty groups. But if it is not used maybe better remove it?

Reply:
Let's remove it.